docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+
+# Connect to an Existing Gno Chain
+
+## 1. Initialize the configurations (required)
+
+```bash
+gnoland config init
+```
+
+## 2. Initialize the secrets (required)
+
+```bash
+gnoland secrets init
+```
+
+## 3. Set a moniker (optional)
+
+```bash
+gnoland config set moniker node02
+```
+
+## 4. Set the rpc connection address (required for connecting with other nodes)
+
+```bash
+gnoland config set rpc.laddr "tcp://0.0.0.0:26657"
+```
+
+## 5. Set the validator private key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_key_file secrets/priv_validator_key.json
+```
+
+## 6. Set the validator state (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set priv_validator_state_file secrets/priv_validator_state.json
+```
+
+## 7. Set the node key (required)
+
+:::tip
+
+The key file path is relative by default.
+
+:::
+
+```bash
+gnoland config set node_key_file secrets/node_key.json
+```
+
+## 8. Obtain the genesis file of the chain to connect to
+
+:::info
+
+The genesis file will be [easily downloadable from GitHub](https://github.com/gnolang/gno/issues/1836#issuecomment-2049428623) in the future.
+
+For now, obtain the file by
+
+1. Sharing via scp or ftp
+2. Getting from `{chain_rpc:26657}/genesis` (might result in time-out error due to large file size)
+
+:::
+
+```bash
+## TODO: Add link to download the file from GitHub
+```
+
+## 9. Add the new validator to existing chain
+
+::: info
+
+This step is currently unavailable. It will be supported in the future after complete implementation of validator set injection with the `r/sys/val` realm.
+
+:::
+
+```bash
+## TODO: Add a new validator
+```
+
+## 10. Confirm the validator information of the first node.
+
+```bash
+# Node ID
+$ gnoland secrets get NodeKey
+
+[Node P2P Info]
+Node ID:  g19d8x6tcr2eyup9e2zwp9ydprm98l76gp66tmd6
+
+# The Public IP of the Node
+$ curl ifconfig.me/ip
+1.2.3.4 # USE YOUR OWN IP
+```
+
+## 11. Configure the persistent_peers list
+
+Configure a list of nodes that your validators will always retain a connection with.
+
+```bash
+$ gnoland config set p2p.persistent_peers "g19d8x6tcr2eyup9e2zwp9ydprm98l76gp66tmd61.2.3.4:26656"
+```
+
+## 12. Configure the seeds
+
+Configure the list of seed nodes. Seed nodes provide information about other nodes for the validator to connect with the chain, enabling a fast and stable initial connection.
+
+:::info
+
+This is an option to configure the node set as the Seed Mode. However, the option to activate the Seed Mode from the node is currently missing.
+
+:::
+
+```bash
+gnoland config set p2p.seeds "g19d8x6tcr2eyup9e2zwp9ydprm98l76gp66tmd6@1.2.3.4:26656"
+```
+
+## 13. Start the second node
+
+```bash
+gnoland chain start
+```
+
+## Results After Starting the Chain and Two Nodes
```

Comment:
This should probably be dropped

Reply:
f93622a3ee8c440e1195cd5f74590b2e69b8322d